### ISSUE TITLE
Python binding fails to buld: RuntimeError: Version not found

### ISSUE THREAD

**Issue by yurivict** [pyomeca/ezc3d#323]
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "setup.py", line 13, in <module>
    raise RuntimeError("Version not found")
RuntimeError: Version not found
*** Error code 1
```

The regular expression in setup.py is inconsistent with the version 1.5.10

Did you actually build the python binding before making a release?

**Comment by pariterre** [comment]
Hi @yurivict !
I was a bit confused when I've seen your message, as I did not even recall that a `setup.py` file was even in the project. This was added by someone else a long time ago. I actually don't even know how to call that file. If you can provide me with the command line you invoked, I can try to debug it though

**Comment by pariterre** [comment]
This is now fixed thanks to #329 (duplicate #328)
Sorry for the delay!
